requirements.txt

```
@@ -5,7 +5,7 @@ canonicalwebteam.blog==6.3.1
 canonicalwebteam.search==0.2.1
 canonicalwebteam.image-template==1.3.1
 canonicalwebteam.store-api==3.0.1
-canonicalwebteam.launchpad==0.7.9
+git+git://github.com/jkfran/canonicalwebteam.launchpad.git@get_snap_build_status#egg=canonicalwebteam.launchpad==0.8.0
```

Waiting on https://github.com/canonical-web-and-design/canonicalwebteam.launchpad/pull/37 to change it to:

```suggestion
canonicalwebteam.launchpad==0.8.0
```